doc/chapters/filesystem/index.rst

```diff
@@ -0,0 +1,102 @@
+Filesystems
+***********
+
+This chapter documents the behavior and layout of the basic filesystem of the
+containers and where it picks up the different packages from. 
+
+
+Standard filesystem
+===================
+
+The base filesystem is bind mounted from a directory created by lxc, the
+directory is empty by default and is setup by the ``lxc-softwarecontainer.in``
```

Comment:
I'd say this is part of understanding softwarecontainer-agent enough to actually use it... you need to understand where the different configuration files come from and what changes the behavior of SCA t o be able to use it properly...